tests/backup/backup_kubevirt_test.go

```diff
@@ -669,3 +669,47 @@ var _ = Describe("{KubevirtUpgradeTest}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{KubevirtVMSshTest}", func() {
+	var (
+		scheduledAppContexts []*scheduler.Context
+	)
+	JustBeforeEach(func() {
+		StartPxBackupTorpedoTest("KubevirtVMSshTest", "Verify SSH to kubevirt VM", nil, 0, Mkoppal, Q1FY25)
+
+		log.InfoD("scheduling applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%d-%d", 93013, i)
+			appContexts := ScheduleApplications(taskName)
+			for _, appCtx := range appContexts {
+				appCtx.ReadinessTimeout = AppReadinessTimeout
+				scheduledAppContexts = append(scheduledAppContexts, appCtx)
+			}
+		}
+	})
+
+	It("Verify backup and restore of Kubevirt VMs in different states", func() {
+
+		Step("Validating applications", func() {
+			log.InfoD("Validating applications")
+			ValidateApplications(scheduledAppContexts)
+		})
+
+		Step("SSH into the kubevirt VM", func() {
+			log.Infof("Sleeping...")
+			//time.Sleep(1 * time.Minute)
+			ctx, err := backup.GetAdminCtxFromSecret()
+			vms, err := GetAllVMsInNamespace(scheduledAppContexts[0].ScheduleOptions.Namespace)
+			if err != nil {
+				return
+			}
+			for _, vm := range vms {
+				log.Infof("Running command for VM [%s]", vm.Name)
+				err = RunCmdInVM(vm, "uname -a", ctx)
```

Comment:
How about?

```go
import "context"

err = RunCmdInVM(vm, "uname -a", context.Background())
```

Reply:
This is just a dummy test case. We can pass `context.Background()` when we write the real test case.